HHS/Head-Start-TTADP
# [Prod] More meaningful duration input, fix bug when approving/creating same report, add background storage cleanup, add migration to remove erroneous records

## Description of change
- Provide better language/improved validation for the duration input on the activity report form

- There was a bug where you'd be shown a weirdly abbreviated form on the review page if you were both a creator and an approver. This has been fixed.

- In an attempt to be proactive, a background process now fires to cleanup locally stored data to hopefully prevent any bugs for users.

- On the CSV export, the objectives would not be de-duplicated by text if it was a multi recipient report, leading to lots of seemingly identical objective columns on the report.

## How to test

- Try to enter in an invalid duration in the activity report form (less than 0, more than 100). Go on. Just try.
- Create a report, mark yourself as an approver, save draft, leave report, come back. Everything should look ok. (In prod, it won't.)
- Export a CSV for a multi-recipient report. You'll notice that the objectives only appear once.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-961
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-828
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-925


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
